htdocs/manage/settings/index.bml

```diff
@@ -445,6 +466,23 @@ body<=
     $ret .= $cats_with_settings{$given_cat}->{desc};
     $ret .= "</p></div>";
 
+    $ret .= "<div class='settings_content'>";
+    $ret .= "<div class='$given_cat'>";
+
+    if ( $given_cat eq "notifications" && $can_view_other ) {
+        # form for viewing another user's notifications
```

Comment:
I know this is necessary here, but every time I see BML I always think "wow, I can't believe this used to be how we made a web site". And still is in this case, but modern Foundation/TT is so much nicer to look at...

And then I wonder what I will think in another 10 years.